app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -39,6 +40,7 @@ class CourseModifyActivity : AppCompatActivity(), CourseModifyContract.View,
     private var isCreated = false
     private var imageNumber: Int = 0
     private var savedImageName: String = ""
+    private var originCourseList = listOf<CourseResponse>()
```

Comment:
```suggestion
    private val originCourseList = mutableListOf<CourseResponse>()
```
val 타입으로 사용하는게 좋지 않을까요?

Reply:
수정했습니당
5780a10